src/Utils/Json.php

```diff
@@ -89,7 +89,7 @@ public static function decode($json, $options = 0)
 		}
 		$value = call_user_func_array('json_decode', $args);
 
-		if ($value === NULL && $json !== '' && strcasecmp($json, 'null')) { // '' is not clearing json_last_error
+		if ($value === NULL && $json !== '' && strcasecmp(trim($json, " \t\n\r"), 'null') != 0) { // '' is not clearing json_last_error
```

Comment:
`strcasecmp()` always returns `int` (or it is supposed so, at least), so there is no difference between using `==` and `===` in this case.